README.md

```diff
@@ -30,47 +32,37 @@ repositories {
 }
 ```
 
-Additionally, add the following Maven repository and (non-sensitive) credentials to your app-level gradle:
+## Usage
 
-```groovy
-repositories {
-    maven {
-        url "https://cardinalcommerceprod.jfrog.io/artifactory/android"
-        credentials {
-            username 'braintree_team_sdk'
-            password 'AKCp8jQcoDy2hxSWhDAUQKXLDPDx6NYRkqrgFLRc3qDrayg6rrCbJpsKKyMwaykVL8FWusJpp'
-        }
-    }
-}
-```
+Create a `DropInRequest` to start the Drop-in UI with specified options:
 
-## Usage
+```kotlin
+val dropInRequest = DropInRequest()
+```
 
-Create a `DropInRequest` and use the `Intent` to start Drop-In:
+`DropInClient` is responsible for launching the Drop-in UI. To launch Drop-in, instantiate a `DropInClient` and call `DropInClient#launchDropInForResult` with the `DropInRequest` you configured above:
 
-```java
-DropInRequest dropInRequest = new DropInRequest()
-    .clientToken(mClientToken);
-startActivityForResult(dropInRequest.getIntent(context), DROP_IN_REQUEST);
+```kotlin
+val dropInClient = DropInClient(this, "<#CLIENT_AUTHORIZATION#>", dropInRequest)
+dropInClient.launchDropInForResult(this, DROP_IN_REQUEST_CODE)
```

Comment:
Should we specify that they need to define this value for `DROP_IN_REQUEST_CODE`

Reply:
Good idea, added a link to those developer docs.

Reply:
Noted that in the description above 👍